toolkit/tools/internal/pkggraph/pkggraph.go

```diff
@@ -1200,12 +1277,15 @@ func (g *PkgGraph) DeepCopy() (deepCopy *PkgGraph, err error) {
 	return
 }
 
+func (g *PkgGraph) MakeDAG() (err error) {
+	return g.MakeDAGusingUpstreamRepos(false, false, nil)
+}
+
 // MakeDAG ensures the graph is a directed acyclic graph (DAG).
```

Comment:
Please update the comment. Looks like this one is left from the original `MakeDAG`.